### DISCUSSION TITLE
Almost default magento-lts setup has a first file '/' time of 0,5s-1s (ttfb) is that similar to yours?

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#1842] [Questions & Answers]
I was wondering what performance your are getting from magento-lts out of the box. I am not interested in bringing this down with plugins/addons or other optimizations. I want to know if my out-of-the-box experience is similar to yours. (Eg. the impact of using my distributed storage)
![developer-magento1-2](https://user-images.githubusercontent.com/13450219/135715268-b24c5919-3d8f-4a63-96e8-6abc37bc6bfc.png)

![developer-magento1-3](https://user-images.githubusercontent.com/13450219/135715273-8187d7d5-c2bd-4cb3-a88d-165ac2fe517b.png)

**Comment by luigifab** [comment]
With latest commit on branch 1.9.4.x, all OpenMage cache enabled, PHP 8.0, MariaDb 10.5, at localhost:

![image](https://user-images.githubusercontent.com/31816829/135845867-595ed0c8-1b4c-43ca-b690-67330bc7d342.png)

**Comment by elidrissidev** [comment]
Localhost the best server provider ![Screenshot from 2021-10-04 21-15-23](https://user-images.githubusercontent.com/67818913/135918589-81f3b2f9-975a-4ea9-9225-897ff49e008c.png)

**Comment by f1-outsourcing** [comment]
Thanks! I was not expecting my 'shitty' connection to have such an influence. (Or actually did not expect to have such shitty connection) ;)
